wcfsetup/install/files/lib/system/registry/RegistryHandler.class.php

```diff
@@ -51,17 +51,17 @@ public function loadStorage(array $packages)
         }
 
         // ignore packages whose storage data is already loaded
-        if (empty($tmp)) {
+        if ($tmp === []) {
             return;
         }
 
         $conditions = new PreparedStatementConditionBuilder();
         $conditions->add("packageID IN (?)", [$tmp]);
 
         $sql = "SELECT  *
-                FROM    wcf" . WCF_N . "_registry
+                FROM    wcf1_registry
                 " . $conditions;
```

Comment:
```suggestion
                {$conditions}";
```
This should be part of [Replace prepareStatement with prepare in RegistryHandler](https://github.com/WoltLab/WCF/pull/5068/commits/bdc25636596e6385bf8188facb4c2d251c611372).